sanic/mixins/listeners.py

```diff
@@ -1,9 +1,12 @@
 from enum import Enum, auto
 from functools import partial
-from typing import Any, Callable, Coroutine, List, Optional, Union
+from sanic.models.handler_types import SignalHandler
+from typing import Any, Callable, Coroutine, List, Optional, TypeVar, Union
 
 from sanic.models.futures import FutureListener
 
+L = TypeVar("L", bound=SignalHandler)
```

Comment:
Why `TypeVar` here? I think you meant `ListenerType`. Also, `L` is not a very descriptive variable name.